proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -106,10 +106,13 @@ message ConsumerRemovalProposal {
 // punish a validator for equivocation on a consumer chain.
 // 
 // This type is only used internally to the consumer CCV module.  
+// WARNING: This message is deprecated now that equivocations can be submitted
+// and verified automatically on the provider. (see SubmitConsumerDoubleVoting in proto/interchain-security/ccv/provider/v1/tx.proto).
```

Comment:
Can we remove those at some point? With a next release or so? If yes, then it would make sense to create an issue so we don't forget to do so.

Reply:
Meaning we should do this if we have a release coming up that needs a migration anyways?